src/tools/fuzzing.h

```diff
@@ -283,6 +284,36 @@ class TranslateToFuzzReader {
         wasm.memory.segments[0].data.push_back(value >= 256 ? 0 : (value & 0xff));
       }
     }
+    // Add memory hasher helper
+    // (for the hash, see hash.h)
+    std::vector<Expression*> contents;
+    contents.push_back(
+      builder.makeSetLocal(0, builder.makeConst(Literal(uint32_t(5381))))
+    );
+    for (Index i = 0; i < USABLE_MEMORY; i++) {
+      // hash = ((hash << 5) + hash) ^ (newbyte & 0xff);
```

Comment:
It would be helpful for me if you could expand this code snippet to include a function signature so I can get the bigger picture right away.

Reply:
what do you mean by "function signature"?

Reply:
I guess I would like to see pseudocode like this:
```
// Inserts the equivalent of the following function:
// int hashMemory() {
//     hash = 0;
//     for each byte of memory...
//         hash = ((hash << 5) + hash) ^ (newbyte & 0xff);
//     return hash;

Reply:
Got it, thanks, adding.